install-dependencies.sh

```diff
@@ -15,7 +15,7 @@ echo "Installing Haskell"
 ./tools/install-stack.sh
 
 echo "Installing Rust"
-curl https://sh.rustup.rs -sSf | sh -s - -y
```

Comment:
I don't think it's working:
```
lryzhyk@nerpa:~/projects/differential-datalog/lib$ rustup default 1.39.0
info: using existing install for '1.39.0-x86_64-unknown-linux-gnu'
info: default toolchain set to '1.39.0-x86_64-unknown-linux-gnu'

  1.39.0-x86_64-unknown-linux-gnu unchanged - rustc 1.39.0 (4560ea788 2019-11-04)

lryzhyk@nerpa:~/projects/differential-datalog/lib$ rustc --version
rustc 1.39.0 (4560ea788 2019-11-04)
lryzhyk@nerpa:~/projects/differential-datalog/lib$ curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.41.1 -y
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
info: updating existing rustup installation


Rust is installed now. Great!

To get started you need Cargo's bin directory ($HOME/.cargo/bin) in your PATH
environment variable. Next time you log in this will be done
automatically.

To configure your current shell run source $HOME/.cargo/env
lryzhyk@nerpa:~/projects/differential-datalog/lib$ rustc --version
rustc 1.39.0 (4560ea788 2019-11-04)
```

Reply:
I don't understand; what is this shell output dump? 
It worked on my machine.

Reply:
This is what the Dockerfile does too

Reply:
maybe on my machine it was another command that did it?

Reply:
I will submit another version

Reply:
The shell output shows that if you have an older version of Rust, this command does not update to 1.41.

Reply:
This is not a problem in Docker, because it install rust on a clean system.